src/Symfony/Component/Notifier/Bridge/AllMySms/AllMySmsTransport.php

```diff
@@ -63,14 +63,31 @@ protected function doSend(MessageInterface $message): SentMessage
 
         $from = $message->getFrom() ?: $this->from;
 
+        $opts = $message->getOptions();
+        $options = $opts ? $opts->toArray() : [];
+        $options['from'] = $options['from'] ?? $from;
+        $options['to'] = $message->getPhone();
+        $options['text'] = $message->getSubject();
+
+        if (isset($options['campaign_name'])) {
+            $options['campaignName'] = $options['campaign_name'];
```

Comment:
why not use `campaignName` key directly in AllMySmsOptions?

Reply:
To main a snake-case standard for all array indexes in the options classes.